### DISCUSSION TITLE
failed to connector

### DISCUSSION THREAD

**Discussion by haiyang1991Liu** [eclipse-edc/Connector#2279] [Q&A]
Hello when I run the comnand./gradlew clean build
it fails and return message Task :spi:common:auth-spi:javadoc FAILED

**Comment by paullatzelsperger** [comment]
tried with the latest `main` - works. what version are you using? And what is the exact error message?

**Comment by wolf4ood** [comment]
@haiyang1991Liu 

did you check on this? probably  you also did hit this regression

https://github.com/eclipse-edc/Connector/discussions/2248
